### ISSUE TITLE
a11y for dropdown list - check and improve

### ISSUE THREAD

**Issue by anqaka** [SnowdogApps/magento2-alpaca-components#253]
check accessibility feature:

- not visible content when in closed
- expanded/collapse attributes
- focus
- keyboard support

**Comment by anqaka** [comment]
internal dev-44082

**Comment by anqaka** [comment]
done on element level
